src/main/scala/mockws/MockWS.scala

```diff
@@ -59,5 +61,19 @@ object MockWS {
     * @param routes       simulation of the external web resource
     * @param materializer user-defined materializer
     */
-  def apply(routes: Routes, materializer: ActorMaterializer) = new MockWS(routes, () ⇒ Unit)(materializer)
+  def apply(routes: Routes, materializer: ActorMaterializer)(implicit fn: RouteNotDefined) = {
+    implicit val mat= materializer
+    new MockWS(routes, () ⇒ Unit){}
+  }
+
+}
+
+trait RouteNotDefined extends (() => Future[Result])
+object RouteNotDefined {
+
+  implicit val defaultAction:RouteNotDefined = RouteNotDefined(NotFound)
```

Comment:
Nitpick: add a whitespace between `defaultAction:` and `RouteNotDefined`

Reply:
We can try to add a `scalafmt` plugin , maybe a new PR after this

Reply:
Here 2d9bb9f02a897ea69f22bdb30d20831046c120e7